cryptography/src/bls12381/benches/partial_signature_aggregation.rs

```diff
@@ -8,7 +8,7 @@ fn benchmark_partial_signature_aggregation(c: &mut Criterion) {
     let msg = b"hello";
     for &n in &[5, 10, 20, 50, 100, 250, 500] {
         let t = quorum(n).unwrap();
-        c.bench_function(&format!("n={} t={}", n, t), |b| {
+        c.bench_function(&format!("partial_aggregate: n={} t={}", n, t), |b| {
```

Comment:
We should also prepend the "dialect" to these. I suppose we really want `[dialect] <name>: arg1=x arg2=y`.

So, this would be `[bls12381] partial_signature_aggregation: n={} t={}`

Reply:
If there is some way to automatically apply any of this, lmk.

We can use this formatting to do some UI magic on the other side.